2312/2/battlers_table.rb

```diff
@@ -0,0 +1,42 @@
+require 'terminal-table'
+
+# BattlersTable prints a table of data, you provide to it
+class BattlersTable
+  def initialize(battlers, num_top)
+    @battlers = battlers
+    @counter = -1
+    @table = []
+    @num_top = num_top
+  end
+
+  def print
+    if @num_top.zero?
```

Comment:
Let's rewrite it a little bit.
```ruby
def print
   terminate_with(message: num_top_zero_message) if num_top.zero?
  
   create_table! # we use bang! because it modifies the object
   puts Terminal::Table.new rows: table[0...num_top]
end
```